standard/docs/en/support/governance.md

```diff
@@ -200,5 +200,15 @@ Anyone who is a current or potential publisher or user of the standard can be co
 
 "The principle of consensus has its origins in the desire to achieve the general acceptance and application of a Standard within its intended sphere of influence. This entails trying to ensure that the interests of all those likely to be affected by it are taken into account, and that individual concerns are carefully and fairly balanced against the wider public interest." [BSI, 2012](http://www.bsigroup.com/Documents/about-bsi/NSB/BSI-pocket-guide-to-standards-development-UK-EN.pdf) 
 
-   
+## Versioning of extensions
+
+Extensions are governed and versioned independently of the standard, with the exception of core extensions, where:
+
+* A MAJOR or MINOR release of a core extension will only take place as part of the governance process for a MAJOR or MINOR release of OCDS. 
```

Comment:
I think it's more future-proof to use the present tense, instead of the future tense, in these paragraphs. We can say, "starting with the version after OCDS 1.1.3" if desired.

Reply:
Done in https://github.com/open-contracting/standard/pull/674/commits/ef99fb58b6213da3d9e35b4b87a54b2a78697990